deploy/helm/kubecf/assets/operations/instance_groups/eirini.yaml

```diff
@@ -103,9 +103,9 @@
           certs_secret_name: eirini-staging-secret
           cc_internal_api: https://cloud-controller-ng.service.cf.internal:9023
           eirini_address: https://{{ .Values.deployment_name }}-eirini.{{ .Release.Namespace }}:8484
-          downloader_image: "eirini/recipe-downloader:0.3.0"
-          uploader_image: "eirini/recipe-uploader:0.3.0"
-          executor_image: "eirini/recipe-executor:0.3.0"
+          downloader_image: "gcr.io/peripli/recipe-downloader:0.3.0-ca-fix"
+          uploader_image: "gcr.io/peripli/recipe-uploader:0.3.0-ca-fix"
+          executor_image: "gcr.io/peripli/recipe-executor:0.3.0-ca-fix"
```

Comment:
Not sure if custom images like these are acceptable. @julz would you consider to prioritize https://www.pivotaltracker.com/story/show/169707308 to make s3 blobstore work without building eirini images with custom fixes?